asmcli/asmcli.sh

```diff
@@ -335,7 +335,7 @@ init() {
 # If DRY_RUN or VERBOSE is enabled, it will print the command, and if DRY_RUN is
 # not enabled it runs the command.
 #######
-run() {
+_run() {
```

Comment:
I made this change because `run` is a `bats` thing to run commands. However, this breaks the cloud build CI. Would need some other way to bypass this.

Reply:
Don't you need to change all occurrences in the entire file?

Reply:
Are you ok with the naming? Should I name it something else?

Reply:
Now that I look at it, it seems like starting identifiers with an underscore isn't valid all the time. Maybe change it to "do" instead?

Reply:
I changed it to `run_commands`.  I'll see if the tests passed. `do` seems to be a keyword in bash?